src/Commands/Import/ImportDatabaseCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Import;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Terminus\Exceptions\TerminusException;
+
+class ImportDatabaseCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    /**
+     * Imports a site archive onto a Pantheon site
+     *
+     * @authorized
+     *
+     * @command database
+     * @name import:database
+     * @aliases import:database
+     *
+     * @option string $site Name of the site to import to
+     * @option string $url  URL at which the import archive exists
+     * @usage terminus import:database <site_name> <archive_url>
+     *   Imports the database in the archive URL to the named site.
+     */
+    public function importDatabase($sitename, $url)
+    {
+        $site = $sitename;
+        list(, $env) = $this->getSiteEnv($site, 'dev');
+        $workflow = $env->importDatabase($url);
+        try {
+            $workflow->wait();
+        } catch (\Exception $e) {
+            if ($e->getMessage() == "Successfully queued import_site") {
```

Comment:
ok - I pulled these out of file and database imports - I'll move it out of Import once I get a clear "move this out of import" from upstream. I'm ok being a peon until then.